common/app/views/support/cleaner/AmpEmbedCleaner.scala

```diff
@@ -198,11 +198,13 @@ case class AmpEmbedCleaner(article: Article) extends HtmlCleaner {
           val linkToInteractive = link.first().attr("href")
           val iframe = document.createElement("amp-iframe")
           val overflowElem = document.createElement("div")
+          val placeholderElem = document.createElement("amp-img")
+
           // In AMP, when using the layout `responsive`, width is 100%,
           // and height is decided by the ratio between width and height.
           // https://www.ampproject.org/docs/guides/responsive/control_layout.html
           iframe.attr("width", "5")
-          iframe.attr("height", "1")
+          iframe.attr("height", "3")
```

Comment:
@Jholder112233 Not sure what is a good starting size for the interactive. What do you think?

If the interactive is out of view from the user, it will resize automatically, but if it is in view, it will stay this size until the user clicks on the "see the full visual" button.